includes/class-wcpdf-assets.php

```diff
@@ -156,6 +156,7 @@ public function backend_scripts_styles ( $hook ) {
 						),
 					),
 					'dismissed_pointers'        => get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true ),
+					'mysql_int_size_limit'      => __( "The typed number is superior to the MySQL allowed INT size, which is 2147483647. Perhaps your number includes the current year? If that's the case you should use the prefix/suffix fields for that.", 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I don't think we need to explain the precise technical limitations. I would go more for something like this:
```
The number should be smaller than 2147483647. Please note you should add your next document number without prefix, suffix or padding. 
```